<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UHFWidget</class>
 <widget class="QWidget" name="UHFWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>193</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>60</y>
     <width>120</width>
     <height>72</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="layout_val">
    <item>
     <widget class="QLabel" name="label_val">
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {font-family:WenQuanYi Micro Hei;font-size:60px;color:green}</string>
      </property>
      <property name="text">
       <string notr="true">-10</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="styleSheet">
         <string notr="true">QLabel {color:white;}</string>
        </property>
        <property name="text">
         <string notr="true">dB</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>35</y>
     <width>221</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget1">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>52</y>
     <width>102</width>
     <height>83</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="layout_max">
    <item>
     <widget class="QLabel" name="label_max">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {color:white;}</string>
      </property>
      <property name="text">
       <string>最大值</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_pluse">
      <property name="styleSheet">
       <string notr="true">QLabel {color:white;}</string>
      </property>
      <property name="text">
       <string>脉冲数</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_degree">
      <property name="styleSheet">
       <string notr="true">QLabel {color:white;}</string>
      </property>
      <property name="text">
       <string>严重度</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
